### PR TITLE
add DVEA

### PR DESCRIPTION
**Pull request by njmulsqb**
add DVEA to the list

### CODE REVIEW

**Comment by njmulsqb** [comment]
I think the references name is expecting a string, and that's the value I have provided. Isn't it?

**Comment by kingthorin** [comment]
It’s expecting one of: https://github.com/OWASP/OWASP-VWAD/blob/bb6e79c6f29bbc18b68a2a137a99b7fb602937ad/schema.json#L106-L114

File: src/data/collection.json [hunk 1]
```diff
@@ -393,6 +393,25 @@
...
+		],
+		"references": [
+			{
+				"name": "Blog Post",
+				"url": "https://njmulsqb.github.io/2023/01/03/releasing-DVEA.html"
+			}
+		],
+		"author": "Najam Ul Saqib (cybersoldier)",
+		"notes": "A deliberately insecure ElectronJS application",
+		"badge": null
```
Comment (kingthorin) [inline src/data/collection.json:413]: Could be `njmulsqb/DVEA`

Comment (njmulsqb) [reply src/data/collection.json:413]: are you talking about badge?

Comment (kingthorin) [reply src/data/collection.json:413]: Yup

**Review by kingthorin — APPROVED** [review]

**Pull request merged**